Core/clim-core/panes.lisp

```diff
@@ -2536,8 +2535,8 @@ SCROLLER-PANE appear on the ergonomic left hand side, or leave set to
    (user-min-height :accessor %pane-user-min-height)
    (user-max-height :accessor %pane-user-max-height)
    ;; size required by the stream
-   (stream-width :initform 0 :accessor stream-width)
-   (stream-height :initform 0 :accessor stream-height))
+   (stream-width :initform 100 :accessor stream-width)
+   (stream-height :initform 100 :accessor stream-height))
```

Comment:
this is good, be we need to fix also 

- `window-clear-method`
 (sets width/height to 0), 

there are other places which change stream width/height size, but imo we should leave them as they are.

Reply:
I looked at that, but was under the (mistaken?) impression that that method was part of the deconstruction of panes. So I assumed it ran before without problems. 
What do we do there? Setting width/height to the 100^2 default is probably a bad idea.

Reply:
I've confused myself. Default sizes require more thought and (maybe) minimal values should be supplied by backends, but window-clear won't be a problem, because frame is already created and compose-space protocol will take care of it.